### PR TITLE
test: add doc test

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

- improve jsdoc
- fix broken links

### CODE REVIEW

**Comment by popomore** [comment]
插件 README 有很多死链

**Comment by fengmk2** [comment]
好吧

File: test/fixtures/apps/docapp/app/middleware/koastatic.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const path = require('path');
+
+module.exports = () => {
+  return require('koa-static')(path.join(process.cwd(), 'run/doctools/public'));
```
Comment (dead-horse) [inline test/fixtures/apps/docapp/app/middleware/koastatic.js]: 怎么不直接用 egg-static？

Comment (popomore) [inline test/fixtures/apps/docapp/app/middleware/koastatic.js]: 不支持 index

**Comment by popomore** [comment]
都得发个版本

**Comment by popomore** [comment]
测试过了就可以了

**Comment by popomore** [comment]
这个可以合了么

**Comment by fengmk2** [comment]
ci得重跑

**Pull request merged**